packages/migrate/src/__tests__/MigrateDiff.test.ts

```diff
@@ -90,22 +90,31 @@ describe('migrate diff', () => {
   })
 
   describe('sqlite', () => {
-    // TODO next 2 tests: is it expected to not fail when diffing from/to an unexisting sqlite db?
-    it('should diff --from-empty --to-url=file:doesnotexists.db', async () => {
+    it('should fail --from-empty --to-url=file:doesnotexists.db', async () => {
       ctx.fixture('schema-only-sqlite')
 
       const result = MigrateDiff.new().parse(['--preview-feature', '--from-empty', '--to-url=file:doesnotexists.db'])
-      await expect(result).resolves.toMatchInlineSnapshot(``)
-      expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`No difference detected.`)
+      await expect(result).rejects.toMatchInlineSnapshot(`
+              P1003
+
+              Database doesnotexists.db does not exist at doesnotexists.db
```

Comment:
Terrible error message btw.

Reply:
The choice of the database file is unfortunate 😅, it's very close to the error message.

The message is more helpful when the path includes a directory below like
```
Database doesnotexists.db does not exist at ./something/doesnotexists.db
```